cocos/platform/apple/CCFileUtils-apple.h

```diff
@@ -54,13 +54,10 @@ class CC_DLL FileUtilsApple : public FileUtils
     virtual bool writeToFile(const ValueMap& dict, const std::string& fullPath) override;
 
     virtual ValueVector getValueVectorFromFile(const std::string& filename) override;
-    void setBundle(NSBundle* bundle);
```

Comment:
My C++ code is like follow, If CCFileUtilsApple has ObjectiveC class, the follow `CryptFileUtils` can compiled:
```C++
#if (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
#include "platform/android/CCFileUtils-android.h"
typedef cocos2d::FileUtilsAndroid FlatformFileUtils;

#elif (CC_TARGET_PLATFORM == CC_PLATFORM_IOS) || (CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
#include "platform/apple/CCFileUtils-apple.h"
typedef cocos2d::FileUtilsApple FlatformFileUtils;

#elif (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
#include "platform/win32/CCFileUtils-win32.h"
typedef cocos2d::FileUtilsWin32 FlatformFileUtils;

#elif (CC_TARGET_PLATFORM == CC_PLATFORM_WINRT)
#include "platform/winrt/CCFileUtilsWinRT.h"
typedef cocos2d::CCFileUtilsWinRT FlatformFileUtils;
//
// ...
#endif


class CryptFileUtils: public FlatformFileUtils {
public:
    CryptFileUtils(Crypt* crypt);
// ...
```